packages/cdk/lib/artifacts/batch-artifacts/ecs-additions/provision.sh

```diff
@@ -62,11 +61,15 @@ ARTIFACT_S3_ROOT_URL=$(\
 
 
 # retrieve and install amazon-ebs-autoscale
-cd /opt
-sh $BASEDIR/get-amazon-ebs-autoscale.sh \
-    --install-version dist_release \
-    --artifact-root-url $ARTIFACT_S3_ROOT_URL \
-    --file-system btrfs
+if [ "$WORKFLOW_ORCHESTRATOR" != "miniwdl" ]; then
+  initial_ebs_size=200
```

Comment:
Do you want to use `INITIAL_SIZE` here?

Reply:
That variable is in the `get-amazon-ebs-autoscale.sh` script which is called by this script. I could however made a parameter for this script that can be initialized.